src/main/webapp/WEB-INF/views/car/addDriverToCar.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add new driver to car</title>
+</head>
+<body>
+<h1>Enter the data to add driver to car</h1>
+<form method="post" action="${pageContext.request.contextPath}/driver/add/car">
+    CAR ID <input type="text" name = "car_id"><br>

```

Comment:
```suggestion
    CAR ID <input type="number" name = "car_id" required><br>

```

Reply:
"required" specifies that an input field must be filled out before submitting the form. Same with Car, Driver and Manufacturer creation